backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -17,7 +18,7 @@
 
 	public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
-	@Query("FROM #{#entityName} q WHERE q.createdAt > (cast(?1 as timestamp)) " +
-			"ORDER BY q.createdAt DESC ")
-	public List<TestEvent> findAllByCreatedAtInstant(String d);
+	// Need to control how this query is built. "between" is too vague.
+	@Query("FROM #{#entityName} q WHERE q.createdAt > ?1 AND q.createdAt <= ?2 ORDER BY q.createdAt DESC ")
```

Comment:
Can we use named parameters here, e.g. `:d1`. Or better yet `:before`, `:after`?